blog/incident-report-week3/index.qmd

```diff
@@ -0,0 +1,30 @@
+---
+author: Mordred Boulais
+title: Incident Report Regarding First Professional Precis
+date: '2023-9-11'
+date-format: YYYY
+categories: [post, developer development, incident report, professional precis, SE1]
+description: <em>A discussion of what we struggled with and how to improve from our first incident.</em>
+---
+
+The overarching struggle with getting out our first blog post regarding `Software
```

Comment:
Please do not reference the book in quotation marks. Instead, it would be better if you give a link to the book itself. This will make it easier for our readers to understand that you are referencing an online book.